WatchFaceAlphaKotlin/app/src/main/java/com/example/android/wearable/alpha/editor/WatchFaceConfigStateHolder.kt

```diff
@@ -212,6 +246,7 @@ class WatchFaceConfigStateHolder(
         if (::editorSession.isInitialized) {
             editorSession.close()
         }
+        scope.cancel()
```

Comment:
It feels a bit weird for `scope.cancel()` to be called here, since the scope is being passed in from above. If something else is in charge of creating `scope`, I would expect that thing to also manage cancelling it.

Reply:
You are right, it is a lifecycleScope... removed.